prime-dotnet-webapi/ApiDbContext.cs

```diff
@@ -0,0 +1,201 @@
+            );
+            #endregion
+
+            #region LicenseSeed
+            modelBuilder.Entity<License>().HasData(
+                new License { Code = 1, Name = "Full - General" },
+                new License { Code = 2, Name = "Full - Pharmacist" },
+                new License { Code = 3, Name = "Full - Specialty" },
+                new License { Code = 4, Name = "Registered Nurse" },
+                new License { Code = 5, Name = "Temporary Registered Nurse" }
+                );
+            #endregion
+
+            #region PracticeSeed
+            modelBuilder.Entity<Practice>().HasData(
+                new Practice { Code = 1, Name = "Remote Practice" },
+                new Practice { Code = 2, Name = "Reproductive Care" },
+                new Practice { Code = 3, Name = "Sexually Transmitted Infections (STI)" },
+                new Practice { Code = 4, Name = "None" }
+                );
+            #endregion
+
+            #region JobNameSeed
+            modelBuilder.Entity<JobName>().HasData(
+                new JobName { Code = 1, Name = "Medical Office Assistant" },
+                new JobName { Code = 2, Name = "Midwife" },
+                new JobName { Code = 3, Name = "Nurse (not nurse practitioner)" },
+                new JobName { Code = 4, Name = "Pharmacy Assistant" },
+                new JobName { Code = 5, Name = "Pharmacy Technician" },
+                new JobName { Code = 6, Name = "Registration Clerk" },
+                new JobName { Code = 7, Name = "Ward Clerk" },
+                new JobName { Code = 8, Name = "Other" }
+                );
+            #endregion
+
+            #region OrganizationNameSeed
+            modelBuilder.Entity<OrganizationName>().HasData(
+                new OrganizationName { Code = 1, Name = "Vancouver Island Health" },
+                new OrganizationName { Code = 2, Name = "Shoppers Drug Mart" }
+                );
+            #endregion
+
+            #region OrganizationTypeSeed
+            modelBuilder.Entity<OrganizationType>().HasData(
+                new OrganizationType { Code = 1, Name = "Health Authority" },
+                new OrganizationType { Code = 2, Name = "Pharmacy" }
+                );
+            #endregion
+
+            #region Indexes
+            modelBuilder.Entity<MailingAddress>()
+                .HasIndex(a => a.EnrolleeId)
+                .HasName("IX_Address_EnrolleeId")
+                .IsUnique(false);
+
+            modelBuilder.Entity<PhysicalAddress>()
+                .HasIndex(a => a.EnrolleeId)
+                .HasName("IX_Address_EnrolleeId")
+                .IsUnique(false);
+
+            modelBuilder.Entity<Address>()
+                .HasIndex("EnrolleeId", "AddressType")
+                .HasName("IX_EnrolleeId_AddressType")
+                .IsUnique(true);
+            #endregion
+
+            #region Relationships
+            // modelBuilder.Entity<MailingAddress>(entity =>
```

Comment:
Is there a reason for the commented out code here?

Reply:
This was accidentally left in the commit - I have removed it now.